config/manager/network_policy.yaml

```diff
@@ -2,11 +2,28 @@
 apiVersion: networking.k8s.io/v1
 kind: NetworkPolicy
 metadata:
-  name: prevent-all-egress
+  name: prevent-all-egress-allow-api-server
   namespace: system
 spec:
   podSelector:
     matchLabels:
-      control-plane: controller-manager
+      control-plane: controller-man
   policyTypes:
     - Egress
+  egress:
+    - to:
+        - ipBlock:
+            cidr: 172.30.0.0/24
```

Comment:
kubeapi ip can be different on different clusters. I'm not sure what the best approach is. But it doesn't seem great to hardcode it here. Or maybe we can have something here, but then patch it using kustomize in the infra repo. And then also customize it using preview.sh when preparing a testing cluster. @davidmogar wdyt?

Reply:
This was just a simple workaround that I asked @naftalysh to test. What I don't get is why we have a PR in here for this. Feature branches and forks are for that. I'm closing it.

Reply:
I can change it to a generic solution